GVFS/GVFS.Common/ProductUpgraderInfo.cs

```diff
@@ -21,19 +21,35 @@ public static string CurrentGVFSVersion()
             return ProcessHelper.GetCurrentProcessVersion();
         }
 
-        public static string GetUpgradesDirectoryPath()
-        {
-            return GVFSPlatform.Instance.GetDataRootForGVFSComponent(UpgradeDirectoryName);
+        public static string GetUpgradeApplicationDirectory()
+        {
+            return Path.Combine(
+                GVFSPlatform.Instance.GetUpgradeProtectedDataDirectory(),
+                ProductUpgraderInfo.ApplicationDirectory);
+        }
+
+        public static string GetParentLogDirectoryPath()

```

Comment:
Can/should this method be removed and replaced with calls to `GVFSPlatform.Instance.GetUpgradeLogDirectoryParentDirectory()` directly?